lmnet/lmnet/datasets/div2k.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import functools
+import os
+from glob import glob
+
```

Comment:
please sort alphabetically

Reply:
Do we adopt [google code style](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_google.py)?
Looking at this [PR](https://github.com/blue-oil/blueoil/pull/455/files), we seemed to adopt the [cryptography style](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_cryptography.py).

Reply:
oh i see. I'm not familiar with the cryptography style but we should do the same as the other blueoil code.